ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -121,12 +118,12 @@ void onBlockImported(ImportedBlockEvent blockImportedEvent) {
     children.forEach(this::importBlockIgnoringResult);
   }
 
-  private void importBlockIgnoringResult(final SignedBeaconBlock block) {
+  public void importBlockIgnoringResult(final SignedBeaconBlock block) {
```

Comment:
I don't think I'd expose this - just have callers use `importBlock` and then the caller should decide how they want to handle the result. I know it will mean a bunch of extra `.reportExceptions()` calls in places that otherwise could use a method reference, but I'm always very nervous about methods that automatically ignore results.

Reply:
Sounds good.